src/common/package.json

```diff
@@ -11,8 +11,10 @@
   "dependencies": {
     "@adobe/commerce-cif-common": "0.1.17",
     "@adobe/commerce-cif-model": "0.1.123",
+    "mkdirp": "^0.5.1",
```

Comment:
Please remove the `^` in the version of packages: we use fixed versions to make sure all the builds are always the same for everybody.